tests/linearbestisorouter_test.py

```diff
@@ -101,11 +55,10 @@ def test_step(self):
 
         path_to_end = res.path + [[*self.track[-1],'']]
         self.assertEqual( res.time, datetime.datetime.fromisoformat('2021-04-02 19:00:00'))
-        self.assertEqual(len(json.dumps(weatherrouting.utils.pathAsGeojson(path_to_end))), 2839)
```

Comment:
this is a very rough way to check if the returned path is the same.
what about checking  geojson identity with md5 hash?

```
import hashlib
md5_res = hashlib.md5(json.dumps(weatherrouting.utils.pathAsGeojson(path_to_end)).encode())
self.assertEqual(md5_res.hexdigest(), 'f5f3ec1f021679c3009865bc162782e0')
```

Reply:
yep, good idea; I'm using sha2 instead of md5